ocs_ci/ocs/platform_nodes.py

```diff
@@ -494,37 +511,50 @@ def restart_nodes(self, nodes, timeout=300, wait=True):
                 and 'ready' state.
 
         """
+        num_events_pre_reboot, num_events_post_reboot = ([] for i in range(2))
         instances = self.get_ec2_instances(nodes)
+        node = nodes[0]
         assert instances, (
             f"Failed to get the EC2 instances for "
             f"nodes {[n.name for n in nodes]}"
         )
+
+        reboot_events_cmd = (
+            f"get events -A --field-selector involvedObject.name="
+            f"{node.name},reason=Rebooted -o yaml"
+        )
+        for node in nodes:
+            num_events_pre_reboot.append(
+                len(node.ocp.exec_oc_cmd(reboot_events_cmd)['items'])
```

Comment:
Modify node name in reboot_events_cmd in each iteration

Reply:
Done